netbox/netbox/views/errors.py

```diff
@@ -49,10 +50,13 @@ def handler_500(request, template_name=ERROR_500_TEMPLATE_NAME):
     except TemplateDoesNotExist:
         return HttpResponseServerError('<h1>Server Error (500)</h1>', content_type='text/html')
     type_, error, traceback = sys.exc_info()
-
+    fname = os.path.split(traceback.tb_frame.f_code.co_filename)[1]
```

Comment:
We may want to expose more of the path than just the filename. If we get an exception in `views.py` for example, the filename alone doesn't help much. We could strip `BASE_DIR` from the file's path and return the remainder.